enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,180 @@
+Blocking these edges for all clusters is also not as aggressive as complaining about an unrecognized 2.0.0 version and failing to serve the entire graph.
+
+### Update service support for the enhanced schema
+
+The following recommendations are geared towards the [openshift/cincinnati][cincinnati].
+Maintainers of other update services may or may not be able to apply them to their own implementation.
+
+The graph-builder's graph-data scraper should learn about [the new 1.1.0 schema](#enhanced-graph-data-schema-for-blocking-edges), and record any `clusters` properties in edge metadata.
+
+When a request is received, the submitted [`channel` query parameter][cincinnati-for-openshift-request] limits the set of remaining edges.
+If any of the remaining edges have `clusters.prometheus` entries, the update service will exclude the edge if the [`id` query parameter][cincinnati-for-openshift-request] is a member of the query's resulting ID set.
+To perform the PromQL request, the update service will be extended with configuration for the Prometheus connection, including a URI and authentication credentials.
+
+To reliably and efficiently return cluster-specific responses, the ID set returned by configured queries may be cached.
+There are a few possibilities for caching; although selecting a particular caching implementation is up to the update service's maintainers.
+The policy engine should refresh the cache on cache misses, as discussed in [the schema section](#enhanced-graph-data-schema-for-blocking-edges).
+
+For every graph-builder response, the policy engine may aggregate queries for all edges and warm the cache of new queries.
+This will make the initial client request faster, at the expense of maintaining a cache that may not be needed before it goes stale.
+
+The policy engine should also be prepared for, and alert on, misconfigured PromQL that results in request failures.
+In the event of such a failure, the fallback behavior should be blocking the relevant edge for all clusters until the misconfiguration is fixed.
+
+### User Stories
+
+#### Bugs which impact a subset of clusters
+
+As described in [the *Motivation* section](#motivation), enabling things like "we'd like to block this edge for clusters born in 4.1 with the `None` platform".
+
+#### OpenShift Dedicated
+
+[OpenShift Dedicated][dedicated] (OSD) restricts the recommended update sets for managed clusters more aggressively than the default service.
+For example, OSD blocked edges into 4.3.19 temporarily while [rhbz#1838007][rhbz-1838007] was investigated, while the default service did not.
+With this enhancement, OSD-specific edge blocking could be accomplished with entries like:
+
+```yaml
+to: 4.3.19
+from: .*
+clusters:
+  # show unmanaged clusters the edge, but block it from managed clusters
+  prometheus: |
+    max by (_id) (0*subscription_labels{managed="true"})
+    or
```

Comment:
maybe the example would look cleaner with `or on() vector(1)` instead of another `max`? That would show "find all managed clusters or return 1" instead of another query?

Reply:
Its not very descriptive though - second part doesn't return `_id`, so it uses "block unknown ids" behaviour

Reply:
I want the cluster to explicitly include blocked-cluster and allowed-cluster IDs.  Hopefully clarified in [the new section](https://github.com/openshift/enhancements/pull/426#discussion_r469417456) , but if not, comment in there.